pyomo/contrib/mindtpy/config_options.py

```diff
@@ -123,59 +84,27 @@ def _get_GDPopt_config():
         description="Stream output to terminal.",
         domain=bool
     ))
-    CONFIG.declare("solver_tee", ConfigValue(
-        default=False,
-        description="Stream the output of mip solver and nlp solver to terminal.",
-        domain=bool
-    ))
     CONFIG.declare("logger", ConfigValue(
         default='pyomo.contrib.mindtpy',
         description="The logger object or name to use for reporting.",
         domain=a_logger
     ))
-    CONFIG.declare("small_dual_tolerance", ConfigValue(
-        default=1E-8,
-        description="When generating cuts, small duals multiplied "
-                    "by expressions can cause problems. Exclude all duals "
-                    "smaller in absolute value than the following."
-    ))
-    CONFIG.declare("integer_tolerance", ConfigValue(
-        default=1E-5,
-        description="Tolerance on integral values."
-    ))
-    CONFIG.declare("constraint_tolerance", ConfigValue(
-        default=1E-6,
-        description="Tolerance on constraint satisfaction."
-    ))
-    CONFIG.declare("variable_tolerance", ConfigValue(
-        default=1E-8,
-        description="Tolerance on variable bounds."
-    ))
-    CONFIG.declare("zero_tolerance", ConfigValue(
-        default=1E-7,
-        description="Tolerance on variable equal to zero."
-    ))
-    CONFIG.declare("initial_feas", ConfigValue(
-        default=True,
-        description="Apply an initial feasibility step.",
-        domain=bool
-    ))
-    CONFIG.declare("obj_bound", ConfigValue(
-        default=1E15,
-        domain=PositiveFloat,
-        description="Bound applied to the linearization of the objective function if master MILP is unbounded."
-    ))
     CONFIG.declare("integer_to_binary", ConfigValue(
         default=False,
-        description="Convert integer variables to binaries (for integer cuts).",
+        description="Convert integer variables to binaries (for no good cuts).",
```

Comment:
are we using `no-good`, `nogood`, or `no good`? Please unify notation across code.